spec/stac-geoparquet-spec.md

```diff
@@ -0,0 +1,39 @@
+# STAC GeoParquet Specification
+
+## Overview
+
+This document specifies how to map a set of [STAC Items](https://github.com/radiantearth/stac-spec/tree/v1.0.0/item-spec) into
+[GeoParquet](https://geoparquet.org). It is directly inspired by the [STAC GeoParquet](https://github.com/stac-utils/stac-geoparquet)
+library, but aims to provide guidance for anyone putting STAC data into GeoParquet. 
+
+## Guidelines
+
+Generally most all the fields in a STAC Item should be mapped to a row in GeoParquet. We embrace Parquet structures where possible, mapping
+from JSON into nested structures. We do pull the properties to the top level, so that it is easier to query and use them. The names of the
+most of the fields should be the same in STAC and in GeoParquet.
+
+| Field           | GeoParquet Type    | Required | Details                                            |
+| --------------- | ------------------ | ---------|--------------------------------------------------- |
+| type            | String             | Optional | This is just needed for GeoJSON, so it is optional and not recommended to include in GeoParquet |
+| stac_extensions | List of Strings    | Required | This column is required, but can be blank if no STAC extensions were used |
+| id              | String             | Required | Required, should be unique |
```

Comment:
I think it's fine to add `within each collection`, but we also have a strong recommendation that each GeoParquet file should only contain _one_ collection, so that the schema is constant across all items.